typescript/lib/parsers/openai.ts

```diff
@@ -601,12 +607,19 @@ export class OpenAIChatModelParser extends ParameterizedModelParser<Chat.ChatCom
             /*accumulatedData*/ message,
             /*index*/ choice.index
           );
+          if (message === undefined) {
+            continue;
+          }
```

Comment:
Same as above. `if (!message)`? or `if (message == null)` (double `==` checks for both null or undefined, triple `===` checks just for null/undefined

Reply:
This is also intentional since message can only be of type `Chat.ChatCompletionMessageParam | undefined` from the `messages.get(choice.index)`. But yea I can add null check too to future-proof

Reply:
I always get worried when I see explicit null or explicit undefined checks. Usually, those are clearly documented when the behavior diverges between null or undefined.

In this case if the behavior between the two is intended to be the same, please switch to `if (!message)`